digdag-core/src/main/java/io/digdag/core/agent/ConfigEvalEngine.java

```diff
@@ -109,7 +113,8 @@ private String invokeTemplate(Invocable templateInvocable, String code, Config p
             throw new TemplateException("Failed to serialize parameters to JSON", ex);
         }
         try {
-            return (String) templateInvocable.invokeFunction("template", code, context);
+            String evaluated = (String) templateInvocable.invokeFunction("template", code, context);
+            return evaluated;
```

Comment:
If there is a significant reason why this change is needed, please add comments here. If it's just styling matter, that's fine.

Reply:
Sorry, I reverted it.